release-team/role-handbooks/ci-signal/README.md

```diff
@@ -133,6 +144,8 @@ The following scalability jobs/tests could regress quite easily (due to seemingl
 - [master-informing gce-scale-performance](https://testgrid.k8s.io/sig-release-master-informing#gce-master-scale-performance)
 - `release 1.xx-blocking gce-cos-1.xx-scalability-100`
 
+Importantly, the expensive scalability jobs that run on master-informing do not run or appear on release-1.x.y-informing because we don't currently run them against the release branch as well as master.  As such, you need to look at these in master-informing even after you've otherwise stopped tracking master, and may need to interpret the delta between master and your release branch.
```

Comment:
These sentences confuse me initially, but I think in the context of CI Signal role handbook it is probably clear enough.  But maybe could use a little additional context or reference to release branch content relative to master especially during code freeze and after code thaw?

Reply:
OK, now you've confused *me*.  I don't understand what "a little additional context or reference to release branch content relative to master especially during code freeze and after code thaw" means.

Reply:
I think what you're describing is that the role involves tracking two branches (release and master), which have different (but similar) content, and have different test automation.  Code flows from master to release branch periodically.  Depending on the test statuses across time, relative to the flow of code content, one can sometimes infer additional release branch health information from master health.  But one can also be deceived.  This is complicated.  I'm not sure the updated text sufficiently uncomplicates it.

Reply:
Oh, no, it's much simpler here.  After branch, the CI Signal role generally doesn't care about master-blocking and master-informing, but only tracks 1.x.0-blocking and 1.x.0-informing.  However, the big performance tests *only* run on master, so those two jobs still have to be tracked there.